src/main/java/org/springframework/retry/support/RetryTemplateBuilder.java

```diff
@@ -497,10 +506,11 @@ public RetryTemplateBuilder withListener(RetryListener listener) {
 	}
 
 	/**
-	 * Appends all provided {@code listeners} to {@link RetryTemplate}'s listener list.
+	 * Appends all provided {@code listeners} to {@link RetryTemplate}'s list of
```

Comment:
The method Javadoc must be imperative: https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting

I see that in some place you have it fixed, but not here.